tests/unifiedPlatform/pds2/pds_dataservice_test.go

```diff
@@ -355,3 +364,71 @@ var _ = Describe("{ValidatePdsHealthIncaseofFailures}", func() {
 		defer EndPDSTorpedoTest()
 	})
 })
+
+var _ = Describe("{RollingRebootNodes}", func() {
+	var (
+		deployment *automationModels.PDSDeploymentResponse
+		err        error
+	)
+
+	JustBeforeEach(func() {
+		StartPDSTorpedoTest("RollingRebootNodes", "Reboot node(s) while the data services will be running", nil, 0)
+	})
+
+	It("Reboot node(s) while the data services will be running", func() {
+		for _, ds := range NewPdsParams.DataServiceToTest {
+			Step("Deploy DataService", func() {
+				deployment, err = WorkflowDataService.DeployDataService(ds, ds.Image, ds.Version, PDS_DEFAULT_NAMESPACE)
+				log.FailOnError(err, "Error while deploying ds")
+				log.Debugf("Source Deployment Id: [%s]", *deployment.Create.Meta.Uid)
+			})
+
+			steplog := "Reboot nodes"
+			Step(steplog, func() {
+				log.InfoD("Reboot nodes")
+				nodesToReboot := node.GetWorkerNodes()
+				for _, n := range nodesToReboot {
```

Comment:
can we write a method for node reboot function, so that we can consume it for other testcases also

Reply:
Done